bridge-stp.in

```diff
@@ -59,27 +114,65 @@ checkpid()
 
 case $2 in
     start)
-        if [ -e "$conf_file" ]; then
-            . "$conf_file" || exit
+        # Make sure the specified bridge is valid.
+        if [ ! -d "$net_dir/$bridge/bridge" ]; then
+            echo "'$bridge' is not a bridge" >&2
+            exit 1
+        fi
+
+        # Determine whether the specified bridge should use MSTP.
+        if ! is_mstp_bridge "$bridge"; then
+            echo "Ignoring bridge '$bridge' that is not listed in \$MSTP_BRIDGES"
+            exit 10
```

Comment:
`exit 10`  ? 
[curios] how should this be treated on exit ?
or does it have some special meaning ?